api/controllers/login.js

```diff
@@ -178,5 +178,16 @@ module.exports = {
       }
       setCookies(req, res, sessionRes);
     });
+  },
+  getIdentity: (req, res) => {
+    res.type('application/json');
+    auth.getUserCtx(req, (err, userCtx) => {
+      if (!err) {
```

Comment:
This logic flow is the other way around compared to the rest of the codebase. I completely missed the `!` when I read it the first time, and wondered why you were replying with a 401 on a successful authentication.

Try flipping this around to `if (err) { .. } ..`